types/progressbar.js/index.d.ts

```diff
@@ -0,0 +1,209 @@
+// Type definitions for progressbar.js 1.1
+// Project: https://kimmobrunfeldt.github.io/progressbar.js/
+// Definitions by: Piotr Błażejewicz <https://github.com/peterblazejewicz>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/progressbar.js/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me